doc/source/whatsnew/v1.5.0.rst

```diff
@@ -832,6 +832,7 @@ I/O
 - :meth:`to_html` now excludes the ``border`` attribute from ``<table>`` elements when ``border`` keyword is set to ``False``.
 - Bug in :func:`read_sas` returned ``None`` rather than an empty DataFrame for SAS7BDAT files with zero rows (:issue:`18198`)
 - Bug in :class:`StataWriter` where value labels were always written with default encoding (:issue:`46750`)
+- Bug in :func:`to_excel` when writing an empty dataframe with row :class:`MultiIndex` (:issue:`19543`)
```

Comment:
This is not a func, rather a method, :meth:`DataFrame.to_excel` and you can remove *row*